buddhi/README.md

```diff
@@ -15,22 +15,28 @@ $ cat traffic.csv
 ## Usage
 
 ```shell
-docker run --rm quay.io/3scale/perftest-toolkit:buddhi-v2-latest -h
+docker run --rm quay.io/3scale/perftest-toolkit:master -h
```

Comment:
Should the users use master floating tag or a fixed tag to use the tool?

I understand this has probably been changed to master because it is merged to master branch. In case users should use a fixed tag how would we make them not download the master tag from the guide?

Reply:
`buddhi-v2-latest` tag was also floating tag. 

I was thinking on creating git release branches and on those stable branches, in doc and ansible tasks, use branch release floating tag. for example:

`quay.io/3scale/perftest-toolkit:v2-latest`

wdyt?

Reply:
Seems good to me :+1: